app/Http/Controllers/Api/AuthController.php

```diff
@@ -60,6 +60,10 @@ public function logout(Request $request) {
         $request->session()->invalidate();
         $request->session()->regenerateToken();
 
-        return $this->returnJson(200, 'User signed out successfully');
+        return $this->returnJson('User logged out successfully', 200);
+    }
+
+    public function session() {
+        return $this->returnJson('User is logged in', 200);
```

Comment:
can you make it return user's `role` on session request? So the response looks somewhat like this:
```
{
    role: 0|1|2
}
```